tiledb/sm/buffer/buffer.h

```diff
@@ -44,18 +43,91 @@ namespace sm {
 
 class ConstBuffer;
 
-/** Enables reading from and writing to a buffer. */
-class Buffer {
+// =======================================================
+//      BufferBase
```

Comment:
Need to change comment in Line 30 above, similar to what we did for the header file.

Reply:
Done.